CardsCashCasino/Data/PokerUtil.cs

```diff
@@ -65,10 +62,13 @@ public enum Ranking
         /// <returns>A 5-card list representing the optimal hand, and the ranking it would give.</returns>
         public static Tuple<List<Card>, Ranking> GetScore(List<Card> community, List<Card> cards)
         {
+            Debug.Assert(cards.Count == 2);
```

Comment:
not _more_ assert calls. i think we need to figure something else out for this but im not cogent enough to identify a solution rn

Reply:
Johnson said preconditions should be represented with assert so i'm using assert